editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -1475,7 +1420,9 @@ void CanvasItemEditor::_viewport_gui_input(const Ref<InputEvent> &p_event) {
 				c = c->get_parent_item();
 		}
 		if (!c) {
-			c = _select_canvas_item_at_pos(click, scene, transform, Transform2D());
+			Vector<_SelectResult> selection;
+			_find_canvas_items_at_pos(click, scene, transform, Transform2D(), selection, 1);
+			if (!selection.empty()) c = selection[0].item;
```

Comment:
Yeah I also prefer that we avoid one-line ifs, they make it harder to distinguish the condition from the actual if block.